stockflux-securities-master/src/components/InputForm.js

```diff
@@ -94,16 +99,21 @@ const InputForm = ({ match }) => {
           });
         })
         .catch(err => {
-          if (err instanceof ValidationError) {
-            setMessages(err.messages);
-          } else {
-            setMessages([err.message]);
-          }
-          setFormState(stateEnum.error);
+          errorHandler(err);
         });
     }
   };
 
+  const onClickDelete = () => {
+    deleteSecurity(match.params.securityId)
+      .then(() => {
+        history.push("/");
+      })
+      .catch(err => {
+        errorHandler(err);
+      });
+  };
+
   return (
```

Comment:
Again, would suggest taking some stuff out of this into their own jsx definitions, because this return is getting very long.

Reply:
We agreed yesterday that this will be addressed in a later refactoring PR. This PR has already included a lot of refactoring.